tests/integration/cots_test.py

```
@@ -1360,6 +1360,21 @@ def test_cots_add_trip_existing_in_navitia():
         assert len(StopTimeUpdate.query.all()) == 0
 
 
+def test_cots_on_add_trip_without_first_cots():
+    """
+    Test that trip update on a newly added trip without first flux cots is rejected with a message
+    """
+    cots_add_file = get_fixture_data('cots_train_151515_added_trip_with_delay.json')
+    res = api_post('/cots', data=cots_add_file)
+    assert res == 'OK'
```

So the response is OK, but the status KO?
To improve that we probably need to raise, not sure.

---

As you said we need to raise an exception to have response status 'KO' and since it's used for gtfs-rt i think it's better as it is